src/docs/guide/animations.md

```diff
@@ -7,4 +7,4 @@ order: 10
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the detailed [animation docs](../core/animation.html) in the meantime.
```

Comment:
I was going to file issues to write these docs; do you want to file issues somewhere? and, if so, in which repo(s)?